src/python_sqs_consumer/sqs_consumer_pool.py

```diff
@@ -18,19 +17,13 @@ def __init__(self, max_threads, processor_class, queue_name):
         self.processor_class = processor_class
         self.queue_name = queue_name
 
-        class _SQSMessagesProcessor(SQSMessagesProcessor):
-            def __init__(self):
-                self.processor_class = processor_class
-
-        self.processor_sqs_class = _SQSMessagesProcessor
-
     def run(self):
         while True:
             for index, worker in enumerate(self.workers_pool):
                 if worker is None or not worker.is_alive():
                     consume = self.consume
                     connect = self.connect
-                    worker = Process(target=consume, args=(connect()))
+                    worker = Process(target=consume, args=(connect(),))
```

Comment:
args param in Process should be a tuple
https://docs.python.org/3/library/multiprocessing.html#the-process-class